### ISSUE TITLE
Prevent lint parsing over most source libraries

### ISSUE THREAD

**Issue by m-tyler** [halcyon-tech/vscode-rpgle#112]
Or it could be stated only allow lint parsing over certain libraries.

My issue is that when I review code in the libraries where mods cannot be made, I end up with lint errors that will not go away until I restart VS Code. 

I have tried adding an empty lint config file and a file with all false but it seems to be picking up another reference for lint rules for that library (my lint config).  

Is there a way to only make the lint parser report issues on source that is opened from my developer library?

**Comment by worksofliam** [comment]
Interesting idea. It's something I will have to think about... is it only indent errors showing up or many types of errors?

**Comment by m-tyler** [comment]
Ah, that makes sense now because that is the issue I see pop up if I open a member from a read-only library source. 

I would not want them turned off for my developer library though.

**Comment by m-tyler** [comment]
This is working as expected.  

I have a question, though.

Is there a cache for the lint rules?  During my testing with lint rules in a library, I don't want rules to be applied, I noticed that changes didn't seem to turn off unless I reloaded VSC.   After waiting what seemed like far too long for the lint rules to update to the marked-up source the errors/warnings finally went away.   I am wondering if there is a quicker way to make the lint rule changes appear in source.
